evm2wast.h

```diff
@@ -0,0 +1,41 @@
+#ifndef __EVM2WAST_H
+#define __EVM2WAST_H
+#include <stdlib.h>
+#include "gadgets.h"
+enum opcodes {
```

Comment:
Is this needed in the external interface?

Reply:
only if you want to access opcodes somewhere after `evm2wast` output

Reply:
Can you remove it in that case?